### ISSUE TITLE
Travis/Carthage not working

### ISSUE THREAD

**Issue by Glennmen** [thepracticaldev/DEV-ios#148]
_Not going to use the bug template because it is not an issue with the iOS app but rather a development pipeline issue so the template didn't apply._

I was looking at this project and noticed that Travis was failing since November 😱

I have done very little iOS development and for sure never worked with `Carthage` but did some searching and I think I found why your Travis builds keep failing.

Carthage uses Github to get the pods from the repositories, but I think we are hitting Github rate limiting because of the shared Travis build servers ip address. To solve this DEV should add a Github API token to their Travis config (encrypted ENV variable).

For reference:
https://github.com/Carthage/Carthage/issues/1236

**Comment by karnett** [comment]
Yup, I also checked this out and needs a GITHUB_ACCESS_TOKEN environment variable. something to do with the 2FA changes GitHub did a few months back.

https://docs.travis-ci.com/user/encryption-keys/
